python/tvm/relay/frontend/onnx.py

```diff
@@ -1050,6 +1050,17 @@ def _impl_v1(cls, inputs, attr, params):
         return AttrCvt("take", extras={"axis": axis})(inputs, {})
 
 
+class GatherElements(OnnxOpConverter):
+    """ Operator converter for GatherElements.
+    """
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        data = inputs[0]
+        indices = inputs[1]
+        axis = attr.get('axis', 0)
```

Comment:
please follow the changes in pr #6448 here also, 
check and change in test code as well.

Reply:
thanks for pointing this out, completely forgot